components/dada-ir/src/span.rs

```diff
@@ -76,6 +76,7 @@ impl Span {
     }
 
     /// Returns a 0-length span at the start of this span
+    #[must_use]
```

Comment:
?

Reply:
It's this one: https://rust-lang.github.io/rust-clippy/master/#return_self_not_must_use

Reply:
Interesting. OK, not opposed. :)